components/Card.js

```diff
@@ -47,3 +53,7 @@ export default function Card({
     </Wrapper>
   );
 }
```

Comment:
Your const Wrapper and your StyledListItem are both li-elements. Here you can use styledComponents with css and define variants

Reply:
it's visually easier for me to have those styled list items named differently, and the wrapper is only used one time. 
Thank you for your hint I will keep it in mind for my other "StyledListItems".